kolibri/plugins/user/assets/src/vue/sign-in-page/index.vue

```diff
@@ -1,16 +1,93 @@
 <template>
 
   <div>
-    <h1>Sign In</h1>
-    <p>Use this page for in-progress component prototyping.</p>
+    <img class="logo" alt="logo">
+    <h1>{{ $tr('signInInstantSchools') }}</h1>
+
+    <form ref="form" @submit.prevent="signIn">
+      <label for="username">{{ $tr('username') }}</label>
+      <input
```

Comment:
Could also just have a method called on focus or keypress that runs a method dedicated to validation? Maybe even an action that changes the local state of the page to show an error